app/uk/gov/hmrc/incorporatedentityidentificationfrontend/models/BusinessVerification.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.incorporatedentityidentificationfrontend.models
+
+import play.api.libs.json.{Json, OFormat}
+
+case class BusinessVerification(verificationStatus: String)
```

Comment:
Could this be an ADT with a trait of BusinessVerificationState then overridden with the possible states?

It's a constant set of states so we could model it as something a little more compiler friendly.

Reply:
Added an ADT